pkg/config/aws/aws.go

```diff
@@ -33,23 +36,39 @@ const (
 	SecretAccessKey = "AWS_SECRET_ACCESS_KEY"
 	// SessionToken represents AWS Session Key
 	SessionToken = "AWS_SESSION_TOKEN"
+
+	assumeRoleDuration = 25 * time.Minute
```

Comment:
For Blockstorage 👆

Reply:
The assumption being that we'll automatically renew the blockstorage creds, right?